examples/build_map_from_scans_and_trajectory.cpp

```diff
@@ -0,0 +1,240 @@
+#include "norlab_icp_mapper/Mapper.h"
+#include <iostream>
+#include <fstream>
+#include <memory>
+#include <sstream>
+#include <vector>
+#include <Eigen/Dense>
+#include <filesystem>
+
+typedef PointMatcher<float> PM;
+typedef PM::DataPoints DP;
+
+namespace fs = std::filesystem;
+
+std::vector<std::pair<Eigen::Affine3d, unsigned long>> getStampedTransformations(const std::string& filepath)
+{
+    // Open the CSV file
+    std::ifstream file(filepath);
+
+    if(!file.is_open())
+    {
+        throw std::runtime_error("Could not open file");
+    }
+
+    // Read the header line to get column indices
+    std::string header;
+    std::getline(file, header);
+
+    std::istringstream header_stream(header);
+    std::vector<std::string> column_names;
```

Comment:
The name of this file and the style of the code inside it is not consistent with the rest of the library (camelCase)

Reply:
I updated the code style however kept the filename to clearly differentiate it from the library files (similarly to libpointmatcher)